### PR TITLE
ruiqu assignment 2 ruby

### PR DESCRIPTION
**Pull request by unata-rae**
tiny instacart will be done in assignment 4; 
the other 2 are ready for review

### CODE REVIEW

File: assignment-2-ruby/ruiqu/customized_room_cleaner.rb [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    mop_floor
+    wash_dishes
+  elsif room == "bathroom"
+    mop_floor
+    clean_toilet
+  else
+    puts "The #{room} requires no specific cleaning processes."
+  end
+
+  mop_floor if room == "kitchen" || room == "bathroom"
```
Comment (thomasfulton) [inline assignment-2-ruby/ruiqu/customized_room_cleaner.rb]: I think you don't need this line as it's covered by the if/else block above?

File: assignment-2-ruby/ruiqu/guessing_game.rb [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        if i == guess
+            puts "Hit! The number is #{i}"
+            return i
+        elsif i > guess
+            last = i - 1
+        elsif i < guess
+            first = i + 1
+        end
+    end
+end
```
Comment (thomasfulton) [inline assignment-2-ruby/ruiqu/guessing_game.rb:78]: Great binary seach!

**Review by thomasfulton — COMMENTED** [review]
Looks good! One small comment about `clean_room` but otherwise 👍

File: assignment-2-ruby/ruiqu/customized_room_cleaner.rb (hunk 1 above)
Comment (unata-rae) [reply assignment-2-ruby/ruiqu/customized_room_cleaner.rb]: right, removed!

**Review by thomasfulton — APPROVED** [review]

**Pull request merged**